### PR TITLE
update promote page and state fact sheets

### PR DESCRIPTION
**Pull request by tanyaparker**
- adds the "Take action in your state" button that currently shows on advocacy.code.org to also show on code.org/promote
- converts the "Code.org's Impact in `state_name`" section in the state facts sheet to a bulleted list
- adds 2 new bullet points to the "Code.org's Impact in `state_name`" section about school and student data

Before (prod numbers) | After (localhost numbers)
------ | ------
image | image

### CODE REVIEW

**Comment by tanyaparker** [comment]
@breville Do you know how to test the state fact sheets on localhost? I tried running `pegasus/rake/generate_pdfs.rake` but it didn't update them locally.

File: pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml [hunk 1]
```diff
@@ -280,80 +280,122 @@ theme: pdf
...
-          teacher
-        - else
-          teachers
-        in Computer Science Principles
-      in
-      = succeed "." do
+            Code.org and
+            = format_integer_with_commas(state_info[:num_affiliates_i])
+            - if state_info[:num_affiliates_i] == 1
+              facilitator have
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml:353]: has?

Comment (breville) [reply pegasus/sites.v3/code.org/public/advocacy/state-facts/splat.haml:353]: Oh, maybe not.

**Comment by tanyaparker** [comment]
Now fixing the last two bullet points that need to be tabbed in.

Before (prod) | After (staging)
------ | ------
image | image

**Pull request merged**